### ISSUE TITLE
When rendered inside an iFrame the modal focus trap prevents the parent iFrame from being focused

### ISSUE THREAD

**Issue by Marcoga** [chakra-ui/chakra-ui#2479]
# Bug report

## Describe the bug

When rendered inside an iFrame the modal focus trap prevents the parent (or sibling) iFrame from being focused. This problem can be observed in this [sample](https://codesandbox.io/s/eager-thunder-c6u7z?file=/src/App.tsx). When you open the modal, you are unable to edit the code in the sandbox.

## To reproduce

1. Go to https://codesandbox.io/s/eager-thunder-c6u7z?file=/src/App.tsx
2. Click on the "Open Modal" button
3. Try to edit the code in the code editor
4. Notice that it is not possible because the focus trap prevents the editor from being focused

## Minimal reproduction

https://codesandbox.io/s/eager-thunder-c6u7z?file=/src/App.tsx

## Expected behavior

react-focus-lock exposes a prop that prevents this from happening. See this [issue](https://github.com/theKashey/react-focus-lock/issues/104). If the FocusLock and Modal components would expose the `crossFrame` prop, the user would be able to opt out of this behavior

## System information

- OS: macOS
- Version of @chakra-ui/react: 1.0.0-rc.8

**Comment by segunadebayo** [comment]
Thanks for this issue and for suggesting a fix.

I've just pushed an update and it'll be added to the next release.

**Comment by with-heart** [comment]
@segunadebayo Glad to see that! This should make development in CodeSandbox quite a bit better 😆
